servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
 ---
 title: ServiceControl in Practice
-summary: Details how to think about the use of ServiceControl in production environments
+summary: Considerations for configuring ServiceControl for use in production environments
```

Comment:
> Optimizing ServiceControl for use in Production

Maybe this is part of what made original attempt at this fail. If we focus on Production only it simplifies the advice.